python/nav/etc/nav.conf

```diff
@@ -51,3 +51,6 @@ DOMAIN_SUFFIX = .example.com
 # Any timestamp NAV generates (using Django) will be relative to this time
 # zone.
 #TIME_ZONE = Europe/Oslo
+
+# Should it be possible to log in with the user set in the header REMOTE_USER?
+#AUTH_SUPPORT_REMOTE_USER = False
```

Comment:
Since this is web specific, and has a sensible default value, it would be more appropriate to have the option in `webfront/webfront.conf`, together with other session and web authentication options.

Reply:
I din't know that file existed. Where is its use/purpose documented?

Reply:
Fixed in "Move config option to other config file"